File: Robo Car Remote Control/carctrl.py
import time
import RPi.GPIO as GPIO
GPIO.setmode(GPIO.BOARD)
GPIO.setup(11, GPIO.OUT)
GPIO.setup(12, GPIO.OUT)

a = GPIO.PWM(11, 50)  # channel=11 frequency=50Hz
b = GPIO.PWM(12, 50)

a.start(0)
b.start(0)

f=open("/var/www/html/carctrl.txt", "r")
while True:
	time.sleep(0.6)
	contents = f.read()
	if (contents == 'B1F\n'):
		a.ChangeDutyCycle(50)
		print('Button 1 Forwards')
	elif (contents == 'B2F\n'):
		b.ChangeDutyCycle(50)
		print('Button 2 Fowards')
	elif (contents == 'F\n'):
		a.ChangeDutyCycle(50)
		b.ChangeDutyCycle(50)
		print('Forwards')
	elif (contents == 'B1FF\n'):
		a.ChangeDutyCycle(100)
		print('Button 1 Fast Fowards')
	elif (contents == 'B2FF\n'):
		b.ChangeDutyCycle(100)
		print('Button 2 Fast Fowards')
	elif (contents == 'FF\n'):
		a.ChangeDutyCycle(100)
		b.ChangeDutyCycle(100)
		print('Fast Forwards')
	elif (contents == 'B1S\n'):
		a.ChangeDutyCycle(0)
		print('Button 1 Stop')
	elif (contents == 'B2S\n'):
		b.ChangeDutyCycle(0)
		print('Button 2 Stop')
	elif (contents == 'S\n'):
		a.ChangeDutyCycle(0)
		b.ChangeDutyCycle(0)
		print('Stop')
	print(contents)
	f.seek(0,0)
GPIO.cleanup()

File: Robo Car Remote Control/carctrl.php
<!DOCTYPE html>
<html>
<body>

<h1>
<?php
if(isset($_GET['F'])){
        $msg = shell_exec("echo F >/var/www/html/carctrl.txt");
        echo "Forward";
}
else if(isset($_GET['B1F'])){
	$msg = shell_exec("echo B1F >/var/www/html/carctrl.txt");
	echo "Button 1 Forward";
}
else if(isset($_GET['B2F'])){
	$msg = shell_exec("echo B2F >/var/www/html/carctrl.txt");
	echo "Button 2 Forward";
}
else if(isset($_GET['FF'])){
        $msg = shell_exec("echo FF >/var/www/html/carctrl.txt");
        echo "Fast Forward";
}
else if(isset($_GET['B1FF'])){
	$msg = shell_exec("echo B1FF >/var/www/html/carctrl.txt");
	echo "Button 1 Fast Forward";
}
else if(isset($_GET['B2FF'])){
	$msg = shell_exec("echo B2FF >/var/www/html/carctrl.txt");
	echo "Button 2 Fast Forward";
}
else if(isset($_GET['S'])){
        $msg = shell_exec("echo S >/var/www/html/carctrl.txt");
        echo "Stop";
}
else if(isset($_GET['B1S'])){
	$msg = shell_exec("echo B1S >/var/www/html/carctrl.txt");
	echo "Button 1 Stop";
}
else if(isset($_GET['B2S'])){
	$msg = shell_exec("echo B2S >/var/www/html/carctrl.txt");
	echo "Button 2 Stop";
}
?>

</h1>




<h2>Basic HTML Table</h2>

<form action="/carctrl.php" method="get" >
	<table border=3 >
	  <tr>
	    	<td>
		<input type="submit" value="B1FF" name="B1FF">
		</td>
	    	<td style="text-align:center">
		<input type="submit" value="Fast_Forward" name="FF">
		</td>
	    	<td>
		<input type="submit" value="B2FF" name="B2FF">
		</td>
	  </tr>
	  <tr>
	    	<td>
		<input type="submit" value="B1F" name="B1F" style="width:100%">
		</td>
	    	<td style="text-align:center">
		<input type="submit" value="Forward" name="F" style="width:100%">
		</td>
	    	<td>
		<input type="submit" value="B2F" name="B2F" style="width:100%">
		</td>
	  </tr>
	  <tr>
	    	<td>
		<input type="submit" value="B1S" name="B1S" style="width:100%">
		</td>
	    	<td style="text-align:center">
		<input type="submit" value="Stop" name ="S" style="width:100%">
		</td>
	    	<td>
		<input type="submit" value="B2S" name="B2S" style="width:100%">
		</td>
	  </tr>
	</table>
</form> 


</body>
</html>


